blog/templates/blog/post_list.html

```diff
@@ -0,0 +1,24 @@
+{% load i18n %}
+
+<html>
+<head>
+    <title>{% trans "Django Girls blog" %}</title>
```

Comment:
I hope you'll introduce template inheritance in upcoming PRs, thus I won't require it now.